ntbs-service/Pages/PreviousHistoryDetails/Edit.cshtml.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using ntbs_service.Helpers;
+using ntbs_service.Models;
+using ntbs_service.Pages;
+using ntbs_service.Services;
+
+namespace ntbs_service.Pages_PatientTBHistory
+{
+    public class EditModel : ValidationModel
+    {
+        private readonly INotificationService service;
+        private readonly NtbsContext _context;

```

Comment:
Let's name this `context` for consistency with the `service` (don't worry about chaning it in other places, I think Farman just has :) )